### ISSUE TITLE
[BUG]Upon closing a drawer scroll, you're unable to scroll.

### ISSUE THREAD

**Issue by troelslenda** [kirbydesign/designsystem#3276]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When you close the drawer, either by pressing backdrop or close button, I'm not able to touch scroll the page for a brief period, while animation finishes. 
 

## Describe how to reproduce the bug
1. Visit https://cookbook.kirby.design/#/home/showcase/modal on a mobile device.
2. open the drawer.
3. close the drawer by clicking close button or backdrop.
4. immediately after (sub 1 second) closing touch the screen to scroll further.
5. experience that scroll is deactivated until you release touch and reengage. 

## Which Kirby version was used?
 
8.11.2 and the one on cookbook.kirby.design

## What was the expected behavior?
That i could scroll earlier.

## Please complete the following information:
- Android 13;SM-A526B Chrome 118.0.5993.111
- iOS 17.1 Edge 109.0.1518.80

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
